### ISSUE TITLE
Does OpenMCU-ru supports mixed protocol rooms?

### ISSUE THREAD

**Issue by alishir** [muggot/openmcu#24]
Does this project support mixed protocol room, for example I create a room named `room291` and SIP client and H.323 could join this room simultaneously.

**Comment by kay27** [comment]
Yes, rooms have high abstraction level, there is no difference between h323 and sip and of course you can use both of them as well as rtsp
